Assets/Scripts/SS3D/Systems/Inventory/Items/ItemSystem.cs

```diff
@@ -39,9 +39,14 @@ private void LoadItemPrefabs()
                 ItemId id = (ItemId)index;
 
                 GameObject itemObject = Assets.Get(id);
-                Item item = itemObject.GetComponent<Item>();
+                ItemActor item = itemObject.GetComponent<ItemActor>();
 
-                item._itemId = id;
+                if (item == null)
+                {
+                    item = new ItemActor();
```

Comment:
I thought you can't instantiate behavior with the new operator ?

Reply:
Just an small error in the code, it was supposed to be new Item();